src/pages/billing/List.js

```diff
@@ -1,27 +1,182 @@
+            value: billingProjectName,
+            onChange: e => this.setState({ billingProjectName: e.target.value, billingProjectNameTouched: true })
+          },
+          error: billingProjectNameTouched && Utils.summarizeErrors(errors && errors.billingProjectName)
+        }),
+        !(billingProjectNameTouched && errors) && Forms.formHint('Name must be unique and cannot be changed.'),
+        Forms.requiredFormLabel('Select billing account'),
+        div({ style: { fontSize: 14 } }, [
+          h(Select, {
+            isMulti: false,
+            placeholder: 'Select billing account',
+            value: chosenBillingAccount,
+            onChange: selected => this.setState({ chosenBillingAccount: selected.value }),
+            options: _.map(account => {
+              return {
+                value: account,
+                label: account.displayName
+              }
+            }, billingAccounts)
+          })
+        ]),
+        !!chosenBillingAccount && !chosenBillingAccount.firecloudHasAccess && div({ style: { fontWeight: 500, fontSize: 12 } }, [
+          div({ style: { color: colors.red[0], margin: '0.25rem 0 0.25rem 0', fontSize: 13 } }, [
+            'Terra does not have access to this account. To grant access, add ', span({ style: { fontWeight: 'bold' } }, 'billing@firecloud.org'),
+            ' as a Billing Account User on the ',
+            a({
+              style: { color: colors.blue[0], fontWeight: 700 },
+              href: `https://console.cloud.google.com/billing/${chosenBillingAccount.accountName.split('/')[1]}?authuser=${Auth.getUser().email}`,
+              target: '_blank'
+            }, ['Google Cloud Console ', icon('pop-out', { size: 12 })])
+          ]),
+          // The following lines will be re-added soon:
+          // div({ style: { marginBottom: '0.25rem' } }, [
+          //   '2. Click ',
+          //   h(Clickable, {
+          //     as: 'span',
+          //     style: { color: colors.blue[0], fontWeight: 700 },
+          //     onClick: () => {
+          //       this.setState({ billingAccounts: undefined })
+          //       this.loadBillingAccounts()
+          //     }
+          //   }, ['HERE']), ' to refresh your billing accounts.'
+          // ]),
+          div({ style: { marginTop: '0.5rem' } }, [
+            'Need help? ',
+            a({
+              style: { color: colors.blue[0], fontWeight: 700 },
+              href: `https://gatkforums.broadinstitute.org/firecloud/discussion/9762/howto-set-up-a-google-billing-account-non-broad-users`,
+              target: '_blank'
+            }, ['Click here ', icon('pop-out', { size: 12 })]), ' for more information.'
+          ])
+        ])
+      ]),
+      submitting && spinnerOverlay
+    ])
+  }
+
+  async submit() {
+    const { onSuccess, ajax: { Billing } } = this.props
+    const { billingProjectName, chosenBillingAccount } = this.state
+
+    try {
+      this.setState({ submitting: true })
```

Comment:
Should probably clear `alreadyExists` here. If the user enters a valid name but some other error happens, the error wouldn't go away unless `alreadyExists` is cleared somewhere.